src/main/java/com/google/songgame/servlets/ChatServlet.java

```diff
@@ -119,16 +128,16 @@ private String getUsername(String userId) {
     return (String) currentUser.getProperty("username");
   }
 
-  // TODO: @salilnadkarni update checkStatus / updateStatus to use Datastore
-  private boolean checkStatus(String userId) {
-    if (!status.containsKey(userId)) {
-      status.put(userId, false);
-    }
-    return status.get(userId) == true;
+  private boolean checkStatus(String userId, Entity currentRound) {
```

Comment:
Is there any way you can be clearer about what a user "status" is. Maybe instead name it "checkGuessedCorrectly."

Reply:
Maybe "checkWonRound", something that indicates what exactly this status is